options/locale/locale_en-US.ini

```diff
@@ -854,7 +854,7 @@ settings.githook_edit_desc = If the hook is inactive, sample content will be pre
 settings.githook_name = Hook Name
 settings.githook_content = Hook Content
 settings.update_githook = Update Hook
-settings.add_webhook_desc = Gitea will send a <code>POST</code> request to the URL you specify, along with regarding the event that occurred. You can also specify what kind of data format you'd like to get upon triggering the hook (JSON, x-www-form-urlencoded, XML, etc). More information can be found in our <a target="_blank" rel="noopener" href="%s">Webhooks Guide</a>.
+settings.add_webhook_desc = Gitea will send a <code>POST</code> request to the URL you specify, along with information about the event that occurred. You can also specify what data format you would like to recieve upon triggering the hook (JSON, x-www-form-urlencoded, XML, etc). More information can be found in our <a target="_blank" rel="noopener" href="%s">Webhooks Guide</a>.
```

Comment:
If you decapitalize "webhooks guide" in settings.hooks_desc you need to do it here also.